### PR TITLE
Updated network description

### PR DESCRIPTION
**Pull request by michaelsena**
WIP, still adding more improvements

### CODE REVIEW

**Comment by michaelsena** [comment]
@oed do you mind taking a look at the above PR for general structure and information hierarchy. If you approve, I will go through and make sure all links work and language is tight. Just added a lot of our libs to the Overview section to provide more clarity. :)

File: README.md [hunk 1]
```diff
@@ -1,62 +1,83 @@
...
 
-## 3Box DB
-3Box DB consists of two components: [`3box-js`](https://www.github.com/3box/3box-js) which is a client library, and [`3box-address-server`](https://www.github.com/3box/3box-address-server) which is a server. Together they are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
+## 3Box Data Network
+The 3Box user data network consists of a few core components: the [pinning server](https://www.github.com/3box/3box-pinning-server), the [address server](https://www.github.com/3box/3box-address-server), and a [caching service](https://www.github.com/3box/3box-caching-service). Together these components are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances on IPFS.
```
Comment (oed) [inline README.md]: The caching service doesn't have it's own repo (`https://www.github.com/3box/3box-caching-service` is linked here). It's built into the pinning-server

File: README.md [hunk 2]
```diff
@@ -1,62 +1,83 @@
...
 
-### 3Box JS
-[`3box-js`](https://www.github.com/3box/3box-js) is the client side library used to read, write, and delete public and private data associated with the user. This library is used to get profile information about an address.
+We assume that end-users have a web3-compatible browser or wallet (such as MetaMask or Status) which supports personal_sign. This is how users authenticate dapps to their 3Box. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
 
-We assume that end users have a web3-compatible browser or wallet (such as MetaMask or Status) which supports personal_sign.
+### 3Box Pinning Server
+[`3box-pinning-server`](https://www.github.com/3box/3box-pinning-server) is a service operated by 3Box that pins 3Box user data on the IPFS network to ensure availability. In the future this service will become more decentralized with a network of pinning nodes, allowing others to run their own node and host their own data if desired.
```
Comment (oed) [inline README.md]: We should add something like: "Right now others can run their own instances of the pinning-server to duplicate the data in the 3box user data network"

File: README.md [hunk 3]
```diff
@@ -1,62 +1,83 @@
...
 
 ### 3Box Address Server
-[`3box-address-server`](https://www.github.com/3box/3box-address-server) is a server utilizing a REST-API that is used to associate an Ethereum address with its latest 3Box address. This is what must be looked up to sync the user's data.    
+[`3box-address-server`](https://www.github.com/3box/3box-address-server) is a server utilizing a REST API that is used to associate an Ethereum address with its 3Box DID (decentralized identifier), to which all user data is addressed. This is what must be looked up to retrieve and sync the user's data. In the future this service will become more decentralized by putting this address to DID mapping on-chain.
```
Comment (oed) [inline README.md]: `by putting this address to DID mapping on-chain.` is not really what we want to do. We want to integrate the address -> DID mapping into the pinning node network.

File: README.md [hunk 4]
```diff
@@ -1,62 +1,83 @@
...
+
+### 3Box Identity
+[`muport-core-js`](https://github.com/3box/muport-core-js) lorem ipsum. Say something about DIDs and associating data to the DID instead of the Ethereum address. This abstraction allows our system to interoperate with other decentralized identity providers and the DIF. DIDs allow multiple Ethereum addresses to be associated with the same 3Box, and allow for other security and usability features such as address rotation that allows users to change their Ethereum address without losing all of their  data.
 
-## 3Box Dapp
+### 3Box Caching Service
```
Comment (oed) [inline README.md]: remove this section

File: README.md [hunk 5]
```diff
@@ -1,62 +1,83 @@
...
-[`3box-dapp`](https://www.github.com/3box/3box-dapp) is a [web interface](https://3box.io/) that allows users to interact with 3Box.
-    
+### 3Box Web App
+[`3box-dapp`](https://www.github.com/3box/3box-dapp) is a [web interface](https://3box.io/) that allows users to interact with their 3Box profile. The 3Box app works with all standard desktop and mobile web3 browsers including MetaMask, Status, Coinbase Wallet, and others.
 
 ### Activity Feed
 [`3box-activity`](https://www.github.com/3box/3box-activity) is an API that makes it easy to construct activity feeds for Ethereum users. Part of Ethereum Profiles on the 3Box web app.
 
+### 3Box Verifications
+[`3box-verifications`](https://github.com/3box/3box-verifications) is a service that allows [3box.io](https://3box.io) users to verify their Github and Twitter social accounts and link them to their 3Box profile.
```
Comment (oed) [inline README.md]: Add something about that the service issues DID compatible verifiable claims.

**Comment by michaelsena** [comment]
Thanks for the comments @oed and @dazuck will integrate that feedback when I have a sec. :+1:

**Comment by oed** [comment]
@michaelsena when you merge this can you make sure to press "Squash and merge"? Right now there is literally a commit for every small little edit.
screenshot 2019-01-10 at 13 02 06

File: README.md [hunk 6]
```diff
@@ -1,62 +1,83 @@
+![3Box Logo](./3Box_widelogo.png)
+
 # 3Box
```
[the remark this answers is not in the document]
Comment (michaelsena) [reply README.md]: If the image doesn't load it's still nice to have the page title especially given this is the main project repo, going to leave for now

**Comment by michaelsena** [comment]
Made all updates, squashing and merging. Woohoo!

**Pull request merged**